dt-core/global-functions.php

```diff
@@ -325,11 +325,18 @@ function dt_render_field_icon( $field, $class = 'dt-icon', $default_to_name = fa
                 } else {
                     $alt_tag = '';
                 }
-                ?>
-
-                <img class="<?php echo esc_html( $class ); ?>" src="<?php echo esc_url( $field['icon'] ) ?>" alt="<?php echo esc_html( $alt_tag ) ?>" width="15" height="15">
 
-                <?php
+                // Determine icon html element shape to be adopted.
```

Comment:
@kodinkat, font icons need to use the `font-icon` field, not the `icon` field, then the code at line 340 works